account_check_deposit_operating_unit/security/check_deposit_security.xml

```diff
@@ -0,0 +1,17 @@
+<odoo>
+    <record id="ir_rule_check_deposit_allowed_operating_units" model="ir.rule">
+        <field
+            name="model_id"
+            ref="account_check_deposit.model_account_check_deposit"
+        />
+        <field
+            name="domain_force"
+        >['|',('operating_unit_id','=',False), ('operating_unit_id','in', user.operating_unit_ids.ids)]</field>
+        <field name="name">Check Deposit from allowed operating units</field>
+        <field name="global" eval="True" />
+        <field eval="0" name="perm_unlink" />
+        <field eval="0" name="perm_write" />
+        <field eval="1" name="perm_read" />
```

Comment:
Readonly is enough?

Reply:
Enough, This is just rule for check operating unit.

Reply:
I mean, user under this rule can create/edit document?

Reply:
Yes, can create/edit document
![Selection_451](https://user-images.githubusercontent.com/24691983/80555669-29d76500-89fb-11ea-9e00-805ba830e739.png)